halo2_proofs/tests/plonk_api.rs

```diff
@@ -478,7 +484,8 @@ fn plonk_api() {
 
         let mut transcript = T::init(vec![]);
 
-        create_plonk_proof::<Scheme, P, _, _, _, _>(
+        create_plonk_proof_with_engine::<Scheme, P, _, _, _, _, _>(
```

Comment:
We also still have the method `halo2_proofs::plonk::create_proof` available.  Did you change this in order to showcase how the engine can be used to create a proof?